src/KeyValidatorTrait.php

```diff
@@ -23,6 +19,12 @@
      */
     protected function validateKey($key)
     {
+        $matched = preg_match('/^[A-Za-z0-9_.]{1,64}$/', $key);
+
+        if (1 !== $matched) {
+            throw new InvalidArgumentException('Can\'t validate the specified key');
```

Comment:
The spec says that the keys are UTF-8 - don't think `{1,64}` will work with multibyte strings. Also this is testing for what MUST be supported, which might be too restrictive depending on the underlying implementation. Would it be better to just test that 1) it's a string and 2) it doesn't contain `{}()/\@:` ?